l10n/en-US/viewer.properties

```diff
@@ -96,9 +96,6 @@ document_properties_subject=Subject:
 document_properties_keywords=Keywords:
 document_properties_creation_date=Creation Date:
 document_properties_modification_date=Modification Date:
-# LOCALIZATION NOTE (document_properties_date_string): "{{date}}" and "{{time}}"
```

Comment:
Please don't remove this, refer to https://github.com/mozilla/pdf.js/pull/10771#discussion_r280981723 for additional details.

Reply:
Fixed in the new commit.